riscv-rt/src/lib.rs

```diff
@@ -177,72 +123,189 @@
 //! REGION_ALIAS("REGION_DATA", RAM);
 //! REGION_ALIAS("REGION_BSS", RAM);
 //! REGION_ALIAS("REGION_HEAP", RAM);
-//! REGION_ALIAS("REGION_STACK", L2_LIM);
+//! REGION_ALIAS("REGION_STACK", RAM);
```

Comment:
`L2_LIM` is a region distinct from main ram. At least on SiFive U74 cores, this is the `SRAM` region.

Reply:
I can rescue the previous example in which `L2_LIM` is used.